compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -1455,6 +1455,11 @@ fn suggest_ampmut<'tcx>(
         && let Ok(src) = tcx.sess.source_map().span_to_snippet(assignment_rhs_span)
         && let Some(stripped) = src.strip_prefix('&')
     {
+        let is_raw_ref = stripped.trim_start().starts_with("raw ");
+        // We don't support raw refs yet
+        if is_raw_ref {
+            return None;
+        }
```

Comment:
Question: I'm assuming by the time here (in MIR borrowck) we lose the detailed spans for the individual components of `&raw const val`, right?

Reply:
Yes that's right. From what I understand from looking at other code it would be better to look up the HIR than to do raw text comparisons. But fixing that would be a big undertaking that I think is better to do at a latter point rather than in this PR.